packages/xo-web/src/common/editable/index.js

```diff
@@ -58,6 +63,7 @@ class Hover extends Component {
   }
 }
 
+// use `data-*` to add params to the handler
```

Comment:
Is it a todo?

Reply:
No, i just want to indicate that the component support `data-*`.